plutus-core/cost-model/test/TestCostModels.hs

```diff
@@ -38,203 +44,255 @@ import           Unsafe.Coerce                                  (unsafeCoerce)
    CostModelCreation to convert R results to picoseconds expressed as
    CostingIntegers.  To deal with (A), we don't check for exact equality of the
    outputs but instead check that the R result and the Haskell result agreee to
-   within a factor of 1/10000 (one hundredth of a percent).
+   within a factor of 1/100 (one percent).
```

Comment:
The accuracy has gone down quite a lot, I think because some of the the numbers are quite small now, so a change in the last digit can be quite significant.